gutenberg.php

```diff
@@ -42,8 +42,7 @@ function the_gutenberg_project() {
 		} else { // Using Gutenberg in Core.
 			printf(
 				// Translators: link is for Classic Editor plugin.
-				__( 'The Block Editor requires JavaScript. Please try the <a href="%s">Classic Editor plugin</a>.', 'gutenberg' ),
-				'https://wordpress.org/plugins/classic-editor/'
+				__( 'The Block Editor requires JavaScript. Please try the <a href="https://wordpress.org/plugins/classic-editor/">Classic Editor plugin</a>.', 'gutenberg' )
```

Comment:
URLs should not be part of translatable strings. We need to use a placeholder for this and make `'https://wordpress.org/plugins/classic-editor/'` translatable separately.

So the correct change would look like this:

```php
printf(
  // Translators: %s: https://wordpress.org/plugins/classic-editor/
  __( 'The Block Editor requires JavaScript. Please try the <a href="%s">Classic Editor plugin</a>.', 'gutenberg' ),
  __( 'https://wordpress.org/plugins/classic-editor/', 'gutenberg' )
);